docs/genai/api/python.md

```diff
@@ -28,6 +28,57 @@ pip install onnxruntime-genai
 import onnxruntime_genai
 ```
 
+## Config class
+
+If the config needs to be modified at runtime to change providers/set provider options, this object can be first created and used, then a model can be created from this config object.
+
+### Load a config
+
+```python
+onnxruntime_genai.Config(model_folder: str) -> onnxruntime_genai.Config
+```
+#### Parameters
+
+- `model_folder`: Location of model and configuration on disk
+
+#### Returns
+
+`onnxruntime_genai.Config`
+
+### Add a provider to the list
+
+If the provider isn't already in the list of providers, this adds it to the end of the list.
```

Comment:
What does adding to the end of the list mean?

Reply:
I tried to find onnxruntime docs about how the provider list works, but couldn't find anything obvious. I don't think we should be explaining it here, do you know where it is?